### DISCUSSION TITLE
Publishing JARs to Github Packages fails due to misconfigured URL?

### DISCUSSION THREAD

**Discussion by triceo** [jreleaser/jreleaser#1430] [General]
I have the following in my `jreleaser.yml` which I then use with JRelease Github Action:

    deploy:
      ...
      github:
        timefold-solver-enterprise:
          active: ALWAYS
          url: "https://maven.pkg.github.com/TimefoldAI/timefold-solver-enterprise/"
      ...

At runtime, I get an error during the very first upload:

    Unexpected error when deploying target/staging-deploy/ai/timefold/solver/enterprise/timefold-solver-enterprise-bom/0.10.0/timefold-solver-enterprise-bom-0.10.0.pom: Got 404 reason: Not Found, The expected resource was not found.

This is an excerpt from the trace log:

    [INFO]      [github]  - timefold-solver-enterprise-bom-0.10.0.pom
    [DEBUG]     [github] url: https://maven.pkg.github.com/TimefoldAI/timefold-solver-enterpriseai/timefold/solver/enterprise/timefold-solver-enterprise-bom/0.10.0/timefold-solver-enterprise-bom-0.10.0.pom

Notice that there is a slash missing between `enterprise` and `ai`, and I'm assuming this leads to the issue. My question is - why would the slash be removed from my config, where it clearly is? Have I misconfigured something?

This is the run in question:
https://github.com/TimefoldAI/timefold-solver-enterprise/actions/runs/5378757873

Trace log is available for download at the very bottom of that page.

**Comment by triceo** [comment]
I found a workaround.

      url: "https://maven.pkg.github.com/TimefoldAI/timefold-solver-enterprise//"

Notice the two slashes at the end. They become one slash, and the upload proceeds.

**Reply by aalmiray** [reply]
Looks like the trailing backslash is removed no matter the case. Will fix this.

**Reply by aalmiray** [reply]
Filed #1431
